site/content/en/docs/tasks/run_rayclusters.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Run A RayCluster"
+date: 2024-01-17
+weight: 6
+description: >
+  Run a RayCluster on Kueue.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [RayCluster](https://docs.ray.io/en/latest/cluster/getting-started.html).
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. By default, the integration for `ray.io/raycluster` is not enabled.
+  Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version)
+  and enable the `ray.io/raycluster` integration.
+
+2. Check [Administer cluster quotas](/docs/tasks/administer_cluster_quotas) for details on the initial Kueue setup.
+
+3. See [KubeRay Installation](https://ray-project.github.io/kuberay/deploy/installation/) for installation and configuration details of KubeRay.
+
+## RayCluster definition
+
+When running [RayClusters](https://docs.ray.io/en/latest/cluster/getting-started.html) on
+Kueue, take into consideration the following aspects:
+
+### a. Queue selection
+
+The target [local queue](/docs/concepts/local_queue) should be specified in the `metadata.labels` section of the RayJob configuration.
```

Comment:
How do we specify `metadata.labels` if we're attempting to create a Ray Job through the CLI (like mentioned below)?

Also sorry for jumping into your PR here - I saw that RayCluster support was added on Friday, so I'm trying it out.